src/datetime/src/components/__snapshots__/DatePicker.test.js.snap

```diff
@@ -221,7 +221,7 @@ exports[`DatePicker render matches snapshot 1`] = `
                 calendarDisabled={Array []}
                 calendarRef={[Function]}
                 calendarValue=""
-                calendarView="2019-05-03"
```

Comment:
Is this going to change for every PR?

Reply:
That's a good question. I suppose I can try and change the test to always select a specific date instead.

Reply:
Oh wait, this prop has to do with highlighting the current day when the calendar is rendered...Yeah not really sure how to handle this situation outside of just removing the snapshot all together. I'll see how atlaskit better tests their component.

Reply:
Yeah, they don't use snapshots
https://bitbucket.org/atlassian/atlaskit-mk-2/src/master/packages/core/datetime-picker/src/internal/__tests__/unit/__snapshots__/index.js.snap